frontend/public/kubevirt/_style.scss

```diff
@@ -1 +1,4 @@
 @import "components/vm";
```

Comment:
This should not be included, as it is part of #17.
The #18 PR will be merged after #16 and #17.